mmedit/edit.py

```diff
@@ -0,0 +1,258 @@
+            Default to None.
+        model_ckpt (str): Path to the checkpoint file for the editing model.
+            Default to None.
+        config_dir (str): Path to the directory containing config files.
+            Default to 'configs/'.
+        device (torch.device): Device to use for inference. Default to 'cuda'.
+    """
+    inference_supported_models = {
+        # conditional models
+        'biggan': {
+            'type': 'conditional',
+            'version': {
+                'a': {
+                    'config':
+                    'biggan/biggan_2xb25-500kiters_cifar10-32x32.py',
+                    'ckpt':
+                    'ckpt/conditional/biggan_cifar10_32x32_b25x2_500k_20210728_110906-08b61a44.pth'  # noqa: E501
+                },
+                'b': {
+                    'config':
+                    'biggan/biggan_ajbrock-sn_8xb32-1500kiters_imagenet1k-128x128.py',  # noqa: E501
+                    'ckpt':
+                    'ckpt/conditional/biggan_imagenet1k_128x128_b32x8_best_fid_iter_1232000_20211111_122548-5315b13d.pth'  # noqa: E501
+                }
+            },
+        },
+
+        # unconditional models
+        'styleganv1': {
+            'type': 'unconditional',
+            'version': {
+                'a': {
+                    'config':
+                    'styleganv1/styleganv1_ffhq-256x256_8xb4-25Mimgs.py',
+                    'ckpt':
+                    'ckpt/unconditional/styleganv1_ffhq_256_g8_25Mimg_20210407_161748-0094da86.pth'  # noqa: E501
+                }
+            }
+        },
+
+        # matting models
+        'gca': {
+            'type': 'matting',
+            'version': {
+                'a': {
+                    'config':
+                    'gca/gca_r34_4xb10-200k_comp1k.py',
+                    'ckpt':
+                    'ckpt/matting/gca/gca_r34_4x10_200k_comp1k_SAD-33.38_20220615-65595f39.pth'  # noqa: E501
+                }
+            }
+        },
+
+        # inpainting models
+        'aot_gan': {
+            'type': 'inpainting',
+            'version': {
+                'a': {
+                    'config':
+                    'aot_gan/aot-gan_smpgan_4xb4_places-512x512.py',
+                    'ckpt':
+                    'ckpt/inpainting/AOT-GAN_512x512_4x12_places_20220509-6641441b.pth'  # noqa: E501
+                }
+            }
+        },
+
+        # translation models
+        'pix2pix': {
+            'type': 'translation',
+            'version': {
+                'a': {
+                    'config':
+                    'pix2pix/pix2pix_vanilla-unet-bn_1xb1-80kiters_facades.py',  # noqa: E501
+                    'ckpt':
+                    'ckpt/translation/pix2pix_vanilla_unet_bn_1x1_80k_facades_20210902_170442-c0958d50.pth'  # noqa: E501
+                }
+            }
+        },
+
+        # restoration models
+        # real_esrgan error
```

Comment:
Wow, what dose this `error` mean?

Reply:
"The model and loaded state dict do not match exactly." I will make a issue to solve this.